koschei/resolver.py

```diff
@@ -311,9 +311,6 @@ def generate_dependency_changes(self, packages, brs, repo_id):
     def run(self, repo_id):
         start = time.time()
         self.log.info("Generating new repo")
-        self.log.info("Polling latest real builds")
-        self.backend.refresh_latest_builds()
-        self.db.commit()
```

Comment:
Could you please split this into separate commit? It's not much related to blocking packages